src/collections/_documentation/platforms/react-native/index.md

```diff
@@ -136,14 +136,44 @@ try {
 }
 ```
 
-## Setting Release / Dist
+### Release Health
+
+Monitor the [health of releases]({%- link _documentation/workflow/releases/health.md -%}) by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases/health.md -%}#crash), and [session data]({%- link _documentation/workflow/releases/health.md -%}#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+To benefit from the health data you must use at least version 1.4.0 of the React Native SDK, and enable the collection of release health metrics when initializing the SDK:
+
+```javascript
+import * as Sentry from '@sentry/react-native';
+    
+Sentry.init({ 
+  dsn: '___PUBLIC_DSN___',
+  enableAutoSessionTracking: true,
+});
+```
+
+The SDK automatically manages the start and end of sessions when the application is started, goes to background, returns to the foreground, etc.
+By default, the session terminates once the application is in the background for more than 30 seconds. To change the timeout, use the option `sessionTrackingIntervalMillis`. For example:
 
 ```javascript
-Sentry.setRelease('release');
-Sentry.setDist('dist');
+import * as Sentry from '@sentry/react-native';
+    
+Sentry.init({ 
+  dsn: '___PUBLIC_DSN___',
+  enableAutoSessionTracking: true,
+  // Sessions close after app is 10 seconds in the background.
+  sessionTrackingIntervalMillis: 10000,
+});
 ```
 
-Since this SDK is unified, all function that are available for JavaScript are also available in this SDK, for more examples:
+For more details, see [full documentation on Release Health]({%- link _documentation/workflow/releases/health.md -%}).
```

Comment:
```suggestion
For more details, see the [full documentation on Release Health]({%- link _documentation/workflow/releases/health.md -%}).
```

Reply:
Oh sorry I missed this. I'll get it done km another PR